src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -130,17 +126,13 @@ export class CardKebabMenuButton extends React.Component<CardKebabMenuButtonProp
         }
     };
 
-    private copyFailureDetails = (event: React.MouseEvent<any>): void => {
+    private copyFailureDetails = async (event: React.MouseEvent<any>): Promise<void> => {
+        const text = this.props.deps.issueDetailsTextGenerator.buildText(this.props.issueDetailsData);
         this.props.deps.detailsViewActionMessageCreator.copyIssueDetailsClicked(event);
 
-        this.props.deps.navigatorUtils
-            .copyToClipboard('The quick brown fox jumps over the lazy dog') // to be changed when we finish the new data format and builder
-            .then(() => {
-                this.setState({ showingCopyToast: true });
-            })
-            .catch(error => {
-                this.logger.error("Couldn't copy failure details!", error);
-            });
+        await this.props.deps.navigatorUtils.copyToClipboard(text);
```

Comment:
Should we still log an error if copying the failure details failed?

Reply:
We are not handing that. Could have a separate PR to inform user, but I don't think that is very valuable.